e2e-test/src/main/java/Test.java

```diff
@@ -0,0 +1,5 @@
+public class Test {
+    public static void main(String[] args) {
+
```

Comment:
What's the purpose of this file?

Reply:
Whenever I would build the jar, it would say that the jar was empty because there was nothing in the main directory so it was confused. It would be a warning locally but when Travis built it, it would be an error and not move forward. I can probably figure out why this is happening and if I can fix it, but I didn't want to hold anything up over it.